bundle/direct/dresources/model.go

```diff
@@ -125,3 +125,11 @@ func (*ResourceMlflowModel) FieldTriggers() map[string]deployplan.ActionType {
 		"tags": deployplan.ActionTypeSkip,
 	}
 }
+
+func (r *ResourceMlflowModel) FieldTriggersLocal() map[string]deployplan.ActionType {
```

Comment:
Looking at this boilerplate in every resource I wonder if we should keep FieldTriggers() support in adapter?  and allow implementations define one of these configurations:
- FieldTriggers
- FieldTriggersLocal & FieldTriggersRemote

Other mixes (e.g. FieldTriggers and FieldTriggersLocal) should be rejected by validate().